Source/WebCore/css/CSSComputedStyleDeclaration.h

```diff
@@ -36,8 +37,12 @@ class MutableStyleProperties;
 class CSSComputedStyleDeclaration final : public CSSStyleDeclaration, public RefCounted<CSSComputedStyleDeclaration> {
     WTF_MAKE_ISO_ALLOCATED_EXPORT(CSSComputedStyleDeclaration, WEBCORE_EXPORT);
 public:
-    WEBCORE_EXPORT static Ref<CSSComputedStyleDeclaration> create(Element&, bool allowVisitedStyle);
-    static Ref<CSSComputedStyleDeclaration> create(Element&, std::optional<PseudoId>);
+    enum class AllowVisited : bool { No, Yes };
+    enum class IsEmpty : bool { No, Yes };
```

Comment:
Nit: presumably this can be private now?

Reply:
AllowVisited is still used outside, but yeah IsEmpty could be private.